ttnn/cpp/ttnn/operations/core/compute_kernel/compute_kernel_config.cpp

```diff
@@ -47,12 +48,14 @@ DeviceComputeKernelConfig init_device_compute_kernel_config(
         if (arch == tt::ARCH::GRAYSKULL) {
             return GrayskullComputeKernelConfig{
                 .math_fidelity = default_fidelity, .math_approx_mode = default_approx_mode};
-        } else {
+        } else if (arch == tt::ARCH::WORMHOLE_B0 || arch == tt::ARCH::BLACKHOLE) {
             return WormholeComputeKernelConfig{
                 .math_fidelity = default_fidelity,
                 .math_approx_mode = default_approx_mode,
                 .fp32_dest_acc_en = default_fp32_acc,
                 .packer_l1_acc = default_l1_acc};
+        } else {
```

Comment:
Please leave this in here. When the next device is brought up there will be a clear error message instead of some random error that will be a waste of time to debug.

Reply:
Or is the plan to have all future architectures use the wormhole compute kernel config, at least as a starting point?

Reply:
I'm not following with new code, if new arch is brought up (like quasar) this piece of code will properly assert?

Reply:
Sorry, you're right.